angular/src/lib/editable-textfield/editable-textfield.component.ts

```diff
@@ -13,18 +13,27 @@ import {
 import { ControlValueAccessor, NgControl } from '@angular/forms';
 
 const cb = () => {};
+
 @Component({
   selector: 'md-editable-textfield',
   template: `
     <div
       *ngIf="isEditing"
       class="md-input-group md-editable-textfield__editing"
-      [ngClass]="textWrapperClasses"
+      [ngClass]="[
+        inputSize ? this.inputSize + ' columns' : '',
+        wrapperClass
```

Comment:
Based on some of the other changes in this PR - why not make this className?

Reply:
fixed